modules/katello_sync.py

```diff
@@ -22,45 +23,41 @@
 module: katello_sync
 short_description: Sync a Katello repository or product
 description:
-    - Sync a Katello repository or product
-author: "Eric D Helms (@ehelms)"
+  - Sync a Katello repository or product
+author:
+  - "Eric D Helms (@ehelms)"
+  - "Matthias M Dellweg (@mdellweg) ATIX AG"
 requirements:
-    - "nailgun >= 0.28.0"
-    - "python >= 2.6"
-    - "ansible >= 2.3"
+  - "nailgun >= 0.28.0"
+  - "ansible >= 2.3"
```

Comment:
I think we should get rid of those ansible requirements :)

Reply:
Not sure, i am following you. Should i just delete line 32?

Reply:
That's what I'd do. And thus only list requirements that are actual python modules a user has to install.

Reply:
I see. Done!